### ISSUE TITLE
Is "Alert Type" useful?

### ISSUE THREAD

**Issue by bbertucc** [bbertucc/equalify#97]
"Alert Type" allows users to filter by errors, warnings, or notices. Those types are added abstractions and, with #96, I wonder if we need Alert Type? 

Currently, WAVE contrast alerts are marked as a warning, failed WCAG 2.1 compliance tests are marked as an error, and system notices are marked as notices.
